hosts/galileo/__init__.py

```diff
@@ -0,0 +1,395 @@
+################################################################################
+# hosts/galileo/__init__.py: Galileo class
+#
+# Utility functions for managing SPICE kernels while working with Galileo data
+# sets.
+################################################################################
+import numpy as np
+import unittest
+import os.path
+
+import julian
+import textkernel
+import spicedb
+import cspyce
+import oops
+
+from oops.body import Body
+
+TOUR = (2003 - 2000) * 365 * 86400      # Rough ET dividing Saturn from Jupiter
+
+################################################################################
+# Routines for managing the loading of C and SP kernels
+################################################################################
+
+# Make sure the leap seconds have been loaded
+oops.spice.load_leap_seconds()
+
+# We load CK and SPK files on a very rough month-by-month basis. This is simpler
```

Comment:
This comment seems left over from Cassini, which had a tour 1997->2017.

Reply:
Yeah, the dates correspond to Cassini.  The approach is the same, so I'll just change the dates.  I don't think it's worth worrying about beyond that since we're going to re-do the spice handling anyway.